web_tool_script_1.R

```diff
@@ -95,6 +95,11 @@ if (new_data == TRUE) {
 ####################
 #### PORTFOLIOS ####
 ####################
+abort_if_file_doesnt_exist(
+  here::here(
+    "working_dir", "20_Raw_Inputs", glue::glue("{portfolio_name_ref_all}.csv")
```

Comment:
Here I see an opportunity to reuse `path_from(raw_inputs(), ...)` from https://github.com/2DegreesInvesting/PACTA_analysis/pull/348

Reply:
why does `abort_if_file_doesnt_exist()` get defined twice in two different scripts?

Reply:
never mind... haven't had my coffee yet ☕